azure-pipelines.yml

```diff
@@ -78,7 +92,7 @@ steps:
   displayName: 'Push NuGet package'
 
 - task: NuGetCommand@2
-  condition: eq(variables.UploadPackage, true)
+  condition: eq(variables['UPLOADPACKAGETOPRERELEASEFEED'], true)
```

Comment:
The same comment as before: `true` should be `'true'` here.

Reply:
Fixed in the latest cherry-pick.